packages/types/src/clerk.ts

```diff
@@ -750,6 +786,32 @@ interface TransferableOption {
 
 export type SignInModalProps = WithoutRouting<SignInProps>;
 
+/**
+ * @experimantal
+ */
+export type __experimental_UserVerificationProps = RoutingOptions & {
+  // TODO(STEP-UP): Verify and write a description
+  afterVerification?: () => void;
+  // TODO(STEP-UP): Verify and write a description
+  afterVerificationUrl?: string;
+
+  /**
+   * Defines the steps of the verification flow.
+   * When `L3.multiFactor` is used, the user will be prompt for a first factor flow followed by a second factor flow.
+   * @default `'L2.secondFactor'`
+   */
+  level?: __experimental_SessionVerificationLevel;
+
+  /**
+   * Customisation options to fully match the Clerk components to your own brand.
+   * These options serve as overrides and will be merged with the global `appearance`
+   * prop of ClerkProvided (if one is provided)
```

Comment:
```suggestion
   * prop of ClerkProvider (if one is provided)
```

Reply:
There seems to be more that one occurrences of this, I'll update the other's as well.